lua/neotest-minitest/init.lua

```diff
@@ -129,6 +129,22 @@ function NeotestAdapter._parse_test_output(output, name_mappings)
   local test_pattern = "(%w+#[%S]+)%s*=%s*[%d.]+%s*s%s*=%s*([FE.])"
   local failure_pattern = "Failure:%s*([%w#_]+)%s*%[([^%]]+)%]:%s*Expected:%s*(.-)%s*Actual:%s*(.-)%s\n\n"
   local error_pattern = "Error:%s*([%w#_]+):%s*(.-)\n[%w%W]-%.rb:(%d+):"
+  local traceback_pattern = "(%d+:[^:]+:%d+:in `[^']+')%s+([^:]+):(%d+):(in `[^']+':[^\n]+)"
```

Comment:
We're capturing the last frame in the stack trace and the last line with the file name, line number and the error message